From axel.lin@gmail.com  Fri Oct 15 12:11:28 2010
Subject: USB: ohci-sh - use resource_size instead of defining its own resource_len macro
From: Axel Lin <axel.lin@gmail.com>
Cc: David Brownell <dbrownell@users.sourceforge.net>,
	Yoshihiro Shimoda <shimoda.yoshihiro@renesas.com>,
	Greg Kroah-Hartman <gregkh@suse.de>, linux-usb@vger.kernel.org
Date: Fri, 15 Oct 2010 13:29:24 +0800
Message-Id: <1287120564.20429.5.camel@mola>

Signed-off-by: Axel Lin <axel.lin@gmail.com>
Acked-by: Yoshihiro Shimoda <yoshihiro.shimoda.uh@renesas.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-sh.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/usb/host/ohci-sh.c
+++ b/drivers/usb/host/ohci-sh.c
@@ -77,7 +77,6 @@ static const struct hc_driver ohci_sh_hc
 
 /*-------------------------------------------------------------------------*/
 
-#define resource_len(r) (((r)->end - (r)->start) + 1)
 static int ohci_hcd_sh_probe(struct platform_device *pdev)
 {
 	struct resource *res = NULL;
@@ -109,7 +108,7 @@ static int ohci_hcd_sh_probe(struct plat
 
 	hcd->regs = (void __iomem *)res->start;
 	hcd->rsrc_start = res->start;
-	hcd->rsrc_len = resource_len(res);
+	hcd->rsrc_len = resource_size(res);
 	ret = usb_add_hcd(hcd, irq, IRQF_DISABLED);
 	if (ret != 0) {
 		err("Failed to add hcd");
